firmware/PinI2C_firmware.h

```diff
@@ -77,11 +77,11 @@
         }
```

Comment:
I did not attempt to figure out and fix the "reset" (i.e. reboot) command. yet.

Reply:
Please don't. What is actually needed is modules cleaning up their internal state back to a clean slate, not a hardware reset. 

My early implementation as hardware reset was out of insecurity concerning how to destroy objects initialized with new() in c++. I've learned OO with a proper language which included garbage collection, and just wanted to push that problem for later. I'll do that soon for the existing modules.

Reply:
I see. I'll skip that.